src/app/portlets/dot-edit-page/content/shared/iframe-edit-mode.css.ts

```diff
@@ -231,11 +235,19 @@ export const getEditPageCss = (timestampId: string): string => {
         text-decoration: none !important;
     }
 
-    ${timestampId} .dotedit-menu__item[title]:hover:after {
-        content: attr(title) !important;
-        left: 100% !important;
+    ${timestampId} .dotedit-menu__item[dot-title]:hover:after {
+        content: attr(dot-title) !important;
+        right: 100% !important;
         position: absolute !important;
-        top: 100% !important;
+        top: 4px !important;
+        background: rgba(0,0,0,.7);
```

Comment:
are these styles only for these specific tooltip or should it be move to a general place, so all tooltips look the same?

Reply:
just specific for this scenario.